pikaur/install_info_fetcher.py

```diff
@@ -616,3 +616,16 @@ def mark_dependent(self) -> None:
                             dep_install_info.provided_by = None
                             dep_install_info.name = name
                             dep_install_info.new_version = dep_install_info.package.version
+
+    def get_total_size(self) -> Dict[str, float]:
```

Comment:
i think it would be looking tidier if changing to get_total_download_size() and  get_total_installed_size()

Reply:
I think about this before, but it seems a little duplicated
now updated

Reply:
thanks!